test/transactionBreakdown.test.ts

```diff
@@ -62,26 +62,57 @@ describe('transactionBreakdown routes', () => {
       let res = await request
         .post('/api/transactionBreakdown')
         .send(transactionBreakdownSeed.transactionBreakdownMissingTax);
+
       expect(res.status).toBe(400);
-      expect(res.text).toBe('Missing content in payload');
+      expect(res.body.message[0].message).toBe('"tax" is required');
 
       res = await request
         .post('/api/transactionBreakdown')
         .send(transactionBreakdownSeed.transactionBreakdownMissingTip);
       expect(res.status).toBe(400);
-      expect(res.text).toBe('Missing content in payload');
+      expect(res.body.message[0].message).toBe('"tip" is required');
 
       res = await request
         .post('/api/transactionBreakdown')
         .send(transactionBreakdownSeed.transactionBreakdownMissingParty);
       expect(res.status).toBe(400);
-      expect(res.text).toBe('Missing content in payload');
+      expect(res.body.message[0].message).toBe('"party" is required');
 
       res = await request
         .post('/api/transactionBreakdown')
         .send(transactionBreakdownSeed.transactionBreakdownMissingItems);
       expect(res.status).toBe(400);
-      expect(res.text).toBe('Missing content in payload');
+      expect(res.body.message[0].message).toBe('"items" is required');
+    });
+
+    it('should throw an error if negative value is sent for tax, tip or item costs', async () => {
```

Comment:
This should be in 3 diff unit tests instead of one if we have three diff responses.

Reply:
Updated the tests and updated the test screenshot for the ticket description accordingly